packages/advisor-components/src/RebootRequired/RebootRequired.tsx

```diff
@@ -0,0 +1,32 @@
+import './RebootRequired.scss';
+
+import React from 'react';
+import { Text, TextContent, TextVariants } from '@patternfly/react-core';
+import { PowerOffIcon } from '@patternfly/react-icons'; // TODO: change PF imports
+import { useIntl } from 'react-intl';
+
+import messages from '../messages';
+
+interface RebootRequiredProps {
+  reboot_required: boolean;
+}
+
+const RebootRequired: React.FC<RebootRequiredProps> = (reboot_required) => {
```

Comment:
Probably a typo, this should be destructed
```suggestion
const RebootRequired: React.FC<RebootRequiredProps> = ({ reboot_required }) => {
```